tests/tests/test-contract-loops.ts

```diff
@@ -101,15 +88,15 @@ describeWithMoonbeam("Moonbeam RPC (Contract Loops)", `simple-specs.json`, (cont
     // 1 loop to make sure it works
     expect(await callLoopIncrContract(1)).to.eq(1);
     let block = await context.web3.eth.getBlock("latest");
-    expect(block.gasUsed).to.eq(42343); //check that gas costs stay the same
+    expect(block.gasUsed).to.eq(42889); //check that gas costs stay the same
 
-    // // 600 loop
-    expect(await callLoopIncrContract(600)).to.eq(600);
+    // // 500 loop
+    expect(await callLoopIncrContract(500)).to.eq(500);
     block = await context.web3.eth.getBlock("latest");
-    expect(block.gasUsed).to.eq(1024084); //check that gas costs stay the same
+    expect(block.gasUsed).to.eq(1045154); //check that gas costs stay the same
 
-    // 700 loop should revert out of gas
-    expect(await callLoopIncrContract(700)).to.eq(0);
+    // 600 loop should revert out of gas
+    expect(await callLoopIncrContract(600)).to.eq(0);
```

Comment:
This is changing the semantics of a test. Was this intentional? Why was it wrong before?

Reply:
I think we didn't have the source code for that contract so we recreated one that doesn't match exactly ?

Reply:
Yes indeed. The new contract is slightly longer and therefore takes more gas, at least it was my understanding